### PR TITLE
Translator action fix

### PR DESCRIPTION
**Pull request by rocketstack-matt**
This should fix the translator GitHub action, it includes a breaking test to validate which must be removed before the merge.

### CODE REVIEW

**Comment by rocketstack-matt** [comment]
This PR fixes the translator build action . . . 
* initial version was checked in with a breaking test to prove it would fail - https://github.com/finos-labs/architecture-as-code/actions/runs/8294926374
* breaking test has now been removed and the action passes - https://github.com/finos-labs/architecture-as-code/actions/runs/8294946581

File: .github/workflows/translator.yml [hunk 1]
```diff
@@ -14,14 +14,10 @@ jobs:
...
-          java-version: '21'
-          distribution: 'temurin'
-          cache: maven
-      - run: mvn -B install --file ./translator/pom.xml
-
-      # Optional: Uploads the full dependency graph to GitHub to improve the quality of Dependabot alerts this repository can receive
-      - name: Submit Dependency Snapshot
-        uses: advanced-security/maven-dependency-submission-action@v3@571e99aab1055c2e71a1e2309b9691de18d6b7d6
\ No newline at end of file
+    - uses: actions/checkout@v4
```
Comment (Budlee) [inline .github/workflows/translator.yml:17]: Is the indentation incorrect here? It looks like it needs an extra tab/space

Comment (rocketstack-matt) [reply .github/workflows/translator.yml:17]: No, it wasn't working previously because the indentation was wrong, now runs.

**Review by Budlee — APPROVED** [review]
@rocketstack-matt let me know that the indentation is correct.
Approve

**Review by matthewgardner — APPROVED** [review]

**Pull request merged**